app/helpers/application_helper.rb

```diff
@@ -15,6 +15,10 @@ def user_logged_in_status
     user_signed_in? ? "logged-in" : "logged-out"
   end
 
+  def user_experience_labels
+    %w[Novice Beginner Mid-level Advanced Expert]
+  end
```

Comment:
Since this is only used in the `users/_ux.html.erb` view, it might make more sense for this to live in its own helper file? Usually when I see a helper in `ApplicationHelper`, it signals to me that it is used throughout the entire app, but since this one is much more specific and is only used in one place, it feels a bit lightweight for this file :) If you did put it in a separate file, you could also extract out the `%w[1 3 5 8 10]` array from [here](https://github.com/thepracticaldev/dev.to/pull/8454/files#diff-46a748ce93f3baf80ce27713e044542cR50), if you wanted! You could also just put this array inline, too.

Not a blocker for merging this PR, though!

Reply:
So I tried putting the array inline but that did not work, erblint kept throwing errors I would try putting it in a separate helper file though

Reply:
I have done the update to separate modules

Reply:
> Since this is only used in the `users/_ux.html.erb` view, it might make more sense for this to live in its own helper file? Usually when I see a helper in `ApplicationHelper`, it signals to me that it is used throughout the entire app, but since this one is much more specific and is only used in one place, it feels a bit lightweight for this file :) If you did put it in a separate file, you could also extract out the `%w[1 3 5 8 10]` array from [here](https://github.com/thepracticaldev/dev.to/pull/8454/files#diff-46a748ce93f3baf80ce27713e044542cR50), if you wanted! You could also just put this array inline, too.
> 
> Not a blocker for merging this PR, though!

@rhymes @vaidehijoshi I have done the update to a separate module, but now I am not sure why Travis is failing.

Reply:
unfortunately it's a flaky spec! Restarted!